src/tests/ftest/scripts/main.sh

```diff
@@ -256,9 +256,20 @@ fi
 # shellcheck disable=SC2086
 export WITH_VALGRIND
 export STAGE_NAME
-# shellcheck disable=SC2086
+
+launch_node_args="-ts ${TEST_NODES}"
+if [ "${STAGE_NAME}" == "Functional Hardware Xlarge" ]; then
```

Comment:
Do we want the "L" capitalized?
```suggestion
if [ "${STAGE_NAME}" == "Functional Hardware XLarge" ]; then
```

Reply:
I do prefer XLarge, but don't want to block on this

Reply:
I was following the syntax of the other stage names which was Uppercase first letter followed by lowercase.

Reply:
Just a minor cosmetic - no worries. If this was following camel case (or most other naming conventions), the L would be capitalized. But again, please don't waste your time re-pushing for this :)